code/modules/reagents/reagent_dispenser.dm

```diff
@@ -392,10 +440,6 @@
 /obj/structure/reagent_dispensers/bloodkeg/cultify()
 	return
 
-/obj/structure/reagent_dispensers/beerkeg/blob_act()
-	explosion(src.loc,0,3,5,7,10)
```

Comment:
You WHAT??

Reply:
I only moved that blob act, not removed it